src/components/accessibility/Listbox.js

```diff
@@ -106,10 +106,9 @@ class Listbox extends React.Component {
     return (
       <div
         aria-label={this.props['aria-label']}
-        className='mx-listbox'
+        className={'mx-listbox' + css({ ...this.props.style })}
```

Comment:
You shouldn't have to spread `this.props.style` in a new object here.  You should be able to just do.

```javascript
className={'mx-listbox' + css(this.props.style)}
```

Reply:
You might even be able to use string interpolation as well

```javascript
className={`mx-listbox ${css(this.props.style)}`}
```